### ISSUE TITLE
A few problems

### ISSUE THREAD

**Issue by loopset** [KaOS-Community-Packages/jdupes#1]
1-You need to add a description to the repository (it may cause problems viewing kcp packages from octopi)
2- I think you should delete the LICENSE file, it no makes sense (you have the line license on the pkgbuild...)

**Comment by demmm** [comment]
Correct the url line too, KaOS has nothing to do with this package, make sure to link to the correct upstream url.
Adding a License file, then not using it anywhere makes no sense, either add & install in the packaging part, or remove.

And as always, **run pckcp** on any you submit, this package is not passing pckcp as is and will be removed if not fixed.

**Comment by kaos-addict** [comment]
Yep sorry for these oversights , shame on me: I'm actually testing some ways to make interacting pckcp and some bash scripts to avoid these kind of problems, it was actually part of my tests and I obviously forgot to delete that repo...
By the way I was wondering about that point: the warning from pckcp is about depends and makedepends variables being empty... But this command doesn't really have deps except for the building stuff. Installing any package that way, is requiring the base-devel group packages, so no need to add it as dep right? Should I let base-devel at least in makedepends?
I removed the license for now I will have a deeper look at the best way to add it... It looks like most of licenses in /usr/share/pkgname/LICENSE link to their /usr/share/doc/pkgname/LICENSE...?

**Comment by demmm** [comment]
base-devel is noted as needed to use KCP (see the guide), so should never be mentioned in makedepends.
Every package "should" include a license, either provide one, or use the tar provided one and install in /usr/share/pkgname/LICENSE
Warnings in pckcp are just that, make sure you check it is actually correct to have no depends.
If you ever want to make sure a package really has no depends run:
```
readelf -d $(pacman -Qql package_name) 2>/dev/null | grep NEEDED | sort | uniq
```
Check if any it links too are not part of base or base-devel.

**Comment by kaos-addict** [comment]
Sure, it's just require libc so I can consider without depends... I will add the license soon.
Thanks for the command line!
